dist/windows/qt.conf

```diff
@@ -2,4 +2,4 @@
 Translations = translations
 
 [Platforms]
-;WindowsArguments = dpiawareness=1
+WindowsArguments = darkmode=2;,dpiawareness=1,fontengine=freetype
```

Comment:
may be use `darkmode=1;` 

> A value of 1 causes Qt to switch the window borders to black when Dark Mode for applications is activated and no High Contrast Theme is in use. This is intended for applications that implement their own theming.

we implement our own theming system

Reply:
@jagannatharjun

I think what they mean in this case is applications that implement their own "forced" themeing (think Winamp/random early 2000s Windows applications). Our themeing is optional, so when no theme is in use, the conditions for `darkmode=2` apply (you can try `darkmode=1`, it does nothing).

Note: I have not yet tested what happens when both a custom theme is in use and `darkmode=2` is in effect. I would expect that the custom theme takes 100% precedence; after all, dark mode on Windows simply changes the base color palette, so it shouldn't matter what color the base color palette is when something goes on top of it.